templates/configmaps.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+data:
+  DB_USER: {{ .Values.mysql.mysqlUser | quote }}
+  DB_HOST: "{{ .Release.Name }}-mysql"
```

Comment:
Here there should be a condition, like if mysql.enabled = true then do this, else get the hostname from values.yaml

Reply:
It is the part of HELM-2(Make Chart configurable with different databases) issue, so I would be pushing that in that pull request

Reply:
okay. Merged! :)